chemprop/data/utils.py

```diff
@@ -127,6 +128,7 @@ def get_data(path: str,
                           in place of :code:`args.features_path`.
     :param features_generator: A list of features generators to use. If provided, it is used
                                in place of :code:`args.features_generator`.
+    :param atom_descriptors_path: The path to the file containing the custom atom descriptors
```

Comment:
Period for consistency.

```suggestion
    :param atom_descriptors_path: The path to the file containing the custom atom descriptors.
```